recipes/qwt/6.2.0/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, get
+from conan.tools.build import cross_building
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.50"
+
+class QwtConan(ConanFile):
+    name = "qwt"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qwt.sourceforge.io/"
+    topics = ("conan", "archive", "compression")
+    description = (
+        "The Qwt library contains GUI Components and utility classes which are primarily useful for programs "
+        "with a technical background. Beside a framework for 2D plots it provides scales, sliders, dials, compasses, "
+        "thermometers, wheels and knobs to control or display values, arrays, or ranges of type double."
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "plot": [True, False],
+        "widgets": [True, False],
+        "svg": [True, False],
+        "opengl": [True, False],
+        "designer": [True, False],
+        "polar": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "plot": False,
+        "widgets": False,
+        "svg": True,
+        "opengl": False,
+        "designer": False,
+        "polar": False,
+        "qt:qtsvg": True
```

Comment:
```suggestion
```
CCI builds only configuration with default values for each options (except for `shared` and `fPIC` options)

Reply:
Without qt:qtsvg enabled by default I get following error:

```
/home/codespace/.conan/data/qwt/6.2.0/_/_/build/b433f42cd4b60c72680e1fed8d58b9efeb5aed8b/src/qwt_symbol.cpp:20:10: fatal error: qsvgrenderer.h: No such file or directory
   20 | #include <qsvgrenderer.h>
```

Indeed in qwt_symbol.cpp we see:

```
#ifndef QWT_NO_SVG
#include <qsvgrenderer.h>
#endif
```

This QWT_NO_SVG variable is specified in qmake pri files.
Then equivalent must be specified in cmake files.